client/me/help/help-search/index.jsx

```diff
@@ -97,23 +86,23 @@ export class HelpSearch extends React.PureComponent {
 		return (
 			<div>
 				<HelpResults
-					header={ this.props.translate( 'WordPress.com Documentation' ) }
+					footer={ translate( 'See more from WordPress.com Documentation…' ) }
+					header={ translate( 'WordPress.com Documentation' ) }
 					helpLinks={ helpLinks.wordpress_support_links }
```

Comment:
These unguarded property accesses concern me. I know they're not new, but we don't have added safety of a transformer in `fromApi` or any additional safety around Redux state.

Reply:
Neither we had any additional safety before 🤷‍♂️ 

But if you insist, I'm happy to add some additional safety (like `_.get()` for example).

Reply:
I'd rather avoid a slew of additional `get` calls on render. A simple `fromApi: makeJsonSchemaParser( { type: 'object' } )` would ensure that it has the expected shape and we'll be able to access properties.

This isn't blocking and I know it's not introduced here.